style/crosswords.less

```diff
@@ -8,40 +8,25 @@
   grid-template-rows: repeat(var(--row-count), var(--grid-cell-size));
   grid-template-columns: repeat(var(--column-count), var(--grid-cell-size));
 
-  input {
```

Comment:
Are the changes here just moves or do they change how it looks? Should I test out this branch and make sure it still looks ok on my app?

Reply:
Hi @mmkal, no behavioural changes intended. Just addressing linting issues and tweaking section order to reflect DOM order.

Please test for behavioural changes!

I'm looking over your PR now. Writing some background for how ids are used in the code!